packages/garbo/src/lib.ts

```diff
@@ -170,6 +171,12 @@ export function expectedEmbezzlerProfit(): number {
 }
 
 export function safeInterrupt(): void {
+  if (
+    globalOptions.prefs.rolloverBuffer * 60 * 1000 <
+    rollover() * 1000 - Date.now()
+  ) {
+    set("garbo_interrupt", true);
```

Comment:
this is real minor, but could you just throw an error saying the reason for the abort instead of setting garbo_interrupt because the garbo_interrupt message isn't technically correct